pallets/subtensor/src/serving.rs

```diff
@@ -263,5 +274,20 @@ impl<T: Config> Pallet<T> {
         return true;
     }
 
+	pub fn validate_axon_data(axon_info: &AxonInfoOf) -> Result<bool, pallet::Error<T>> {
+		if axon_info.port <= 0 {
```

Comment:
:warning: If the [internet][1] is to be believed, then the highest port supported by IPv4 addresses _should_ be `65535`

While it be true that `port` for `AxonInfo` is correctly defined as `u16` (`pallets/subtensor/src/lib.rs:319`) I'm not fully certain of how things will behave if someone feeds a larger number during execution.

[1]: https://stackoverflow.com/questions/113224/what-is-the-largest-tcp-ip-network-port-number-allowable-for-ipv4#113228

Reply:
Will check for overflow, expect a commit momentarily.

Reply:
Resolved